bucket/git-archive-all.json

```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://github.com/Kentzo/git-archive-all",
+    "license": "MIT",
+    "version": "1.19.2",
+    "url": "https://github.com/Kentzo/git-archive-all/archive/1.19.2.zip",
+    "hash": "386d095d4b462b29cb177959e15210569e9fd78f9dffb7d1cb7129645ae8facf",
+    "extract_dir": "git-archive-all-1.19.2",
+    "depends": "python",
+    "post_install": [
+        "pushd \"$dir\"",
+        "try {",
+        "    scoop reset python",
```

Comment:
Why this reset??

Reply:
The formula is based on `https://github.com/lukesampson/scoop/blob/master/bucket/git-up.json` and the trick is borrowed there. My idea was to use an existing recipe for the identical installation process.

Reply:
Reset is nonsense and could cause problems to users, which don't want to have actual python in path or whatever. 

It's already in try block, so doing `python --version | Out-Null` should be enough.

Reply:
Add catch block with some message like `No python found` or something like that

Reply:
What if there is a non-scoop Python in user's PATH? Or this situation is not possible (i.e. scoop overrides PATH to avoid such conflicts)?

Reply:
That's reason why this should not depend on python. It force installing new python (Not preserving use-case when user have other python installed already).

So best way possible is move that try block inside pre_install (in catch block write message and exit with non zero code) and then keep installer block with only setup.py and context changes.

Reply:
Isn't it expected for software (in general) installed from scoop to depend on scoop-managed software? I don't want to make assumption about the kind of Python installed by the user (e.g. too old version).

The script gets installed into Python's Scripts directory, therefore it's necessary to ensure that the python executable is scoop's Python.